src/components/presentations/UserProfile/UserProfile.jsx

```diff
@@ -5,9 +5,6 @@ import Headercard from '../HeaderCard/Headercard';
 import Profilecard from './ProfileCard';
 import Reportcard from './ReportedCard';
 import SuggestedArticleCard from './SuggestedArticleCard';
-import dummyData from '../../../utils/dummyData';
-
-const { profileReport } = dummyData;
 
 class Userprofile extends Component {
   constructor(props) {
```

Comment:
You can remove this constructor @sojida since you are no longer initializing anything inside it. Our ESLint settings should be flagging this 🤔

Reply:
Eslint would flag it if no constructor is initialized.